controllers/argocd/openshift/openshift.go

```diff
@@ -285,7 +285,7 @@ func policyRulesForClusterConfig() []rbacv1.PolicyRule {
 			},
 		}, {
 			APIGroups: []string{
-				"machineconfig.openshift.io",
+				"machineconfiguration.openshift.io",
```

Comment:
Hi @sejo , Thanks for the PR.

Is there a change in the API group name in the recent versions or It has always been "machineconfiguration.openshift.io" ?

Reply:
Actually as of 4.1 I only see references to machineconfiguration.openshift.io in the official documentation.